packages/mongodb/src/mongodb-store.service.spec.ts

```diff
@@ -427,6 +454,123 @@ describe('MongoDBStore', () => {
 
     });
 
+    describe('has a "getAuthenticatedUserIds" method that', () => {
+
+      beforeEach(async () => {
+        await createSessionTestData();
+      });
+
+      it('should return all distinct user ids', async () => {
```

Comment:
Maybe we should add another test or extend this one here to test the case that, when a user is not authenticated (i.e `userId === null`), the method does not return a `null` value in its returned array.

Reply:
I'll add that.